services/src/main/proto/census.proto

```diff
@@ -0,0 +1,318 @@
+// Copyright 2016, Google Inc.
```

Comment:
What's the relationship of this census.proto w/ the monitoring.proto?  They don't seem to actually be related, at least not on the surface, does it make sense to put them in as the same commit or split them up?